README.md

```diff
@@ -118,6 +118,14 @@ variable. Use this if you'd like to customize the display of org
 notifications when using a daemon such as
 [dunst](https://github.com/knopwob/dunst).
 
+### Custom categories
+
+org-alert uses the category `'org-alert` by default. You can set this to
```

Comment:
Could you run `fill-paragraph` on this? Maybe I have a weird setup and it's already formatted, but it shifted things around a bit when I ran it.